docs/cli/kops_edit.md

```diff
@@ -5,7 +5,7 @@ Edit clusters and other resources.
 ### Synopsis
 
 
-Edit a resource configuration. This command changes the cloud specification in the registry. 
+Edit a resource configuration. This command changes the cloud configuration in the registry. 
```

Comment:
Updated as proposed. "Desired configuration" is really the same as "specification" but it avoids confusion.